### DISCUSSION TITLE
How can I find the zoomed state of a slide? zoomChange event fires inconsistantly.

### DISCUSSION THREAD

**Discussion by inquiztr** [nolimits4web/swiper#5684] [Q&A]
I want to synchronize the zoomed state with other elements on my page (display an icon etc.)
I tried monitoring the state in the 'zoomChange' event by looking if the slide has the zoomed class but I found the 'zoomChange' event fires at different lifecycle of updating the css class names.

The 'zoomChange' event fires after 'swiper-slide-zoomed' class is added to the slide'
The 'zoomChange' event fires before 'swiper-slide-zoomed' is about to be removed'.

swiperGallery.on('zoomChange', function (a, b, c, d) {
            // will always appear slide is zoomed because class always exists
            console.log('hasclass:' + d.classList.contains('swiper-slide-zoomed'));
}

if I can't rely on CSS class, is there another way, perhaps an internal property I could access?

**Comment by dinhkham** [comment]
check scale when zoomed.
Check swiper.zoom.scale === 1 with reactJs
